script.js

```diff
@@ -0,0 +1,234 @@
+  `;
+  sectioncard5.appendChild(main);
+  sectioncard5.appendChild(maincard);
+
+  const closeButton2 = document.querySelector('.close-button2');
+  closeButton2.addEventListener('click', () => {
+    sectioncard5.removeChild(maincard);
+    sectioncard5.removeChild(main);
+  });
+});
+
+const sectioncard6 = document.querySelector('.project-6');
+const openerButton6 = document.querySelector('.button6');
+openerButton6.addEventListener('click', () => {
+  const main = document.createElement('div');
+  main.className = 'main';
+  const maincard = document.createElement('div');
+  maincard.className = 'maincard';
+  maincard.innerHTML = `
+  <button class="close-button2">&times;</button>
+  <div>
+ 
+  <img  class = "popup-image"src="images/pop-up/Snapshoot Portfolio.png" alt="">
+ </div>
+       <div class ="frame18"> <button class="first-button1" type="button" id="messagep">See source <img src="images/pop-up/button-icons/github.png" alt=""></button>
+        <button class="first-button2" type="button" id="messagep">See live <img src="images/pop-up/button-icons/Icon.png" alt=""></button>
+    <h3 class="popup-par">Keeping track of hundreds of <br> components</h3></div>
+    <ul class="popup-list">
+      <li class="ruby">Codekit</li>
+      <li class="css">Github</li>
+      <li class="javascript">JavaScript</li>
+      <li class="html">Bootstrap</li>
+      <li class="html">Terminal</li>
+      <li class="html">Codepen</li>
+    </ul>
+    <div class ="lorem"><h2>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea</h2></div>
+  `;
+  sectioncard6.appendChild(main);
+  sectioncard6.appendChild(maincard);
+
+  const closeButton2 = document.querySelector('.close-button2');
+  closeButton2.addEventListener('click', () => {
+    sectioncard6.removeChild(maincard);
+    sectioncard6.removeChild(main);
+  });
+});
+
+const contactForm = document.getElementById('form-id');
+const contactEmail = document.getElementById('email');
+const contactError = document.getElementById('errormessage');
+
+contactForm.addEventListener('submit', (event) => {
+  if (contactEmail.value.toLowerCase() !== contactEmail.value) {
+    event.preventDefault();
+    contactError.innerText = 'please type your email in lowercase.form not submitted';
```

Comment:
Your error message is empty 🗑️  at first but when you set the `innerText` it is always `absolute` so it doesn't show up 👀 you can fix this regardless of my other comment by adding this line below:
```
contactError.style.position = 'inherit';
```
This just gives the `div` the right position attribute to make it show up near the fields. 😉